helpers/data-exchange-payload-generator.js

```diff
@@ -0,0 +1,135 @@
+
+
+export class DataExchangePayloadGenerator {
+    itemsAmount
```

Comment:
this is not required to create the variable by default (and in other places as well)